clang/test/CodeGen/aarch64-sme2-intrinsics/acle_sme2_mlal.c

```diff
@@ -460,7 +460,7 @@ void test_svmla_single4_u16(uint32_t slice_base, svuint16x4_t zn, svuint16_t zm)
 //
 void test_svmla_single4_s16(uint32_t slice_base, svint16x4_t zn, svint16_t zm) __arm_streaming __arm_shared_za
 {
-   SVE_ACLE_FUNC(svmla_single_za32,,_s16,,_vg2x4)(slice_base, zn, zm);
+   SVE_ACLE_FUNC(svmla,_single,_za32,_s16,_vg2x4)(slice_base, zn, zm);
 }
```

Comment:
The tests for the `svmla_lane` builtins in this file also need to be updated.

Reply:
Since `SVE_ACLE_FUNC` is defined as `SVE_ACLE_FUNC(A1,A2_UNUSED,A3,A4_UNUSED,A5) A1##A3##A5` The svmla_lane test
`SVE_ACLE_FUNC(svmla_lane_za32,,,_s16,_vg2x4)(slice_base, zn, zm, 7);`
is functionally the same as
`SVE_ACLE_FUNC(svmla_lane_za32,_s16,_vg2x4,,,)(slice_base, zn, zm, 7);`
which is why I didn't update these tests earlier. Do we still want the commas to be pushed to the back?

Reply:
It was trivial to change them so I just went ahead and did it.